tst.so/src/tst_worker.cpp

```diff
@@ -154,7 +154,7 @@ void TSTWorker::blasThread(int gpuIdx, uint64_t matrix_size, std::string tst_ops
     gem_ops = 0;
    // setup rvsBlas
     gpu_blas = std::unique_ptr<rvs_blas>(new rvs_blas(gpuIdx,  matrix_size,  matrix_size,  matrix_size, transa, transb, alpha, beta, 
-          tst_lda_offset, tst_ldb_offset, tst_ldc_offset, tst_ops_type));
+          tst_lda_offset, tst_ldb_offset, tst_ldc_offset, tst_ops_type, ""));
```

Comment:
instead of this, we can have default param for type as empty string in prototype of rvs_blas?

Reply:
We will have to still pass complete set of arguments to the rvs_blas constructor Right ?